resource-watch/resource-watch
# Updates legend timeline styles

## Overview
Updates legend timeline styles according to feedback received.
With the new version of the timeline, the default value will be calculated based on the active layer, instead of the first one as it was working before.

## Testing instructions

- Run `yarn` to install latest `vizzuality-components` version, should be `1.3.0`.

- Open http://localhost:9000/data/explore?zoom=4&lat=12.55456352859367&lng=14.23828125&basemap=dark&labels=light&layers=%255B%257B%2522dataset%2522%253A%2522595bcf6f-0343-4146-ba0d-c54b1c928510%2522%252C%2522opacity%2522%253A1%252C%2522layer%2522%253A%2522d131bcf4-f1ba-41b6-94bc-0099236e719b%2522%257D%255D&page=1&sort=most-viewed&sortDirection=-1&topics=%255B%2522population%2522%255D to see a dataset with timeline configuration.

Before:
![image](https://user-images.githubusercontent.com/999124/59267226-ef7da180-8c49-11e9-9262-3e9c1e7062ce.png)

After:
![image](https://user-images.githubusercontent.com/999124/59868648-b7850580-9391-11e9-8f4c-bd1cbe8779b7.png)



## Pivotal task
Part of: https://www.pivotaltracker.com/story/show/164251672

---

## Checklist before submitting
